satpy/etc/readers/viirs_vgac_l1c_nc.yaml

```diff
@@ -0,0 +1,253 @@
+reader:
+  name: viirs_vgac_l1c_nc
+  short_name: VGAC from VIIRS MOD and IMG channels
+  long_name: VIIRS Global Area Coverage from VIIRS Reflected Solar Band and Thermal Emission Band data for both Moserate resolution and Imager resolution channels.
+  description: >
+    NetCDF4 reader for VGAC from VIIRS.
```

Comment:
Do you have a link or reference for data corresponding to this reader?  I know the compact format and the SDR format.  I've also heard that EUMETSAT Secretariat are considering to distribute new global data via EUMETCAST, but I don't know the format of the latter.  Is that VGAC?

Reply:
I am not sure if this is the data to be distributed by EUMETCAST. Here is some information https://ams.confex.com/ams/JOINTSATMET/mediafile/Handout/Paper360556/VGAC%20version%201%20-%20poster.pdf